<?xml version="1.0"?>
<launch>

  <!-- Joint velocity and acceleration estimator -->
  <node name="left_arm_joint_state_estimator" pkg="joint_state_estimator" type="joint_state_estimator_node" cwd="node" respawn="false" output="screen" >
    <rosparam command="load" file="$(find joint_state_estimator)/config/dumbo_left_arm_joint_state_estimator_config.yaml" />
    <remap from="/left_arm_joint_state_estimator/state" to="/left_arm_controller/state"/>
  </node>

</launch>